stablehlo/testdata/BUILD.bazel

```diff
@@ -28,7 +28,7 @@ expand_template(
         "\"@STABLEHLO_SOURCE_DIR@\"": "os.path.join(os.environ['TEST_SRCDIR'], '_main')",
         "\"@STABLEHLO_TOOLS_DIR@\"": "os.path.join(os.environ['TEST_SRCDIR'], '_main')",
         "@LIT_SITE_CFG_IN_HEADER@": "# Autogenerated, do not edit.",
-        "@LLVM_TOOLS_DIR@": package_path("@llvm-project//llvm:BUILD"),
+        "@LLVM_TOOLS_DIR@": "../llvm-project/llvm",
```

Comment:
I filed https://github.com/llvm/llvm-project/issues/83066 -- this cuts down on the # of symlinks effectively by half.